zeppelin-zengine/src/test/resources/test-zeppelin-site3.xml

```diff
@@ -0,0 +1,167 @@
+  </property>
+
+  <property>
+    <name>zeppelin.notebook.storage</name>
+    <value>org.apache.zeppelin.notebook.repo.S3NotebookRepo</value>
+    <description>notebook persistence layer implementation</description>
+  </property>
+  -->
+
+  <property>
+    <name>zeppelin.notebook.storage</name>
+    <value>org.apache.zeppelin.notebook.repo.VFSNotebookRepo</value>
+    <description>notebook persistence layer implementation</description>
+  </property>
+
+  <property>
+    <name>zeppelin.interpreter.dir</name>
+    <value>interpreter</value>
+    <description>Interpreter implementation base directory</description>
+  </property>
+
+  <property>
+    <name>zeppelin.interpreter.connect.timeout</name>
+    <value>30000</value>
+    <description>Interpreter process connect timeout in msec.</description>
+  </property>
+
+
+  <property>
+    <name>zeppelin.ssl</name>
+    <value>false</value>
+    <description>Should SSL be used by the servers?</description>
+  </property>
+
+  <property>
+    <name>zeppelin.ssl.client.auth</name>
+    <value>false</value>
+    <description>Should client authentication be used for SSL connections?</description>
+  </property>
+
+  <property>
+    <name>zeppelin.ssl.keystore.path</name>
+    <value>keystore</value>
+    <description>Path to keystore relative to Zeppelin configuration directory</description>
+  </property>
+
+  <property>
+    <name>zeppelin.ssl.keystore.type</name>
+    <value>JKS</value>
+    <description>The format of the given keystore (e.g. JKS or PKCS12)</description>
+  </property>
+
+  <property>
+    <name>zeppelin.ssl.keystore.password</name>
+    <value>change me</value>
+    <description>Keystore password. Can be obfuscated by the Jetty Password tool</description>
+  </property>
+
+  <!--
+  <property>
+    <name>zeppelin.ssl.key.manager.password</name>
+    <value>change me</value>
+    <description>Key Manager password. Defaults to keystore password. Can be obfuscated.</description>
+  </property>
+  -->
+
+  <property>
+    <name>zeppelin.ssl.truststore.path</name>
+    <value>truststore</value>
+    <description>Path to truststore relative to Zeppelin configuration directory. Defaults to the keystore path</description>
+  </property>
+
+  <property>
+    <name>zeppelin.ssl.truststore.type</name>
+    <value>JKS</value>
+    <description>The format of the given truststore (e.g. JKS or PKCS12). Defaults to the same type as the keystore type</description>
+  </property>
+
+  <property>
+    <name>zeppelin.server.allowed.origins</name>
+    <value>http://onehost:8080,http://otherhost.com,</value>
+    <description>Allowed sources for REST and WebSocket requests.</description>
+  </property>
+  <property>
+    <name>zeppelin.notebook.public</name>
+    <value>true</value>
+    <description>Make notebook public by default when created, private otherwise</description>
+  </property>
+  <property>
+    <name>zeppelin.notebook.owner</name>
```

Comment:
One suggestion. Instead of setting admin as one of owners, what about bypassing privilege check in case of admin? 

This approach doesn't modify existing notebooks and backward compatible.